src/components/Welcome/Welcome.tsx

```diff
@@ -1,14 +1,23 @@
-
-import { Anchor, Text, Title } from '@mantine/core';
-import classes from './Welcome.module.css';
+import { Text, Title, useMantineColorScheme } from '@mantine/core';
 import { useTranslation } from 'react-i18next';
+import { welcomeStyles } from './styles';
+import { useMediaQuery } from '@mantine/hooks';
 
 export function Welcome() {
   const { t } = useTranslation();
+  const { colorScheme } = useMantineColorScheme();
+  const isMobile = useMediaQuery('(max-width: 768px)');
+
+  const titleStyle = {
```

Comment:
Isnt this the default component that came with the repo template? why are you changing this ?